core/org.openjdk.jmc.flightrecorder/src/main/java/org/openjdk/jmc/flightrecorder/stacktrace/StacktraceModel.java

```diff
@@ -417,6 +417,7 @@ public int compare(FrameEntry o1, FrameEntry o2) {
 		private final int itemOffset;
 		private final int itemsInFork;
 		private Integer selectedBranchIndex;
+		private final SimpleArray<IItem> allItems;
```

Comment:
This file to update the copyright year to 2021

Reply:
Done. Thanks for pointing it out and approving the pull request.